common/locales/en/settings.json

```diff
@@ -41,7 +41,7 @@
     "json": "(JSON)",
     "customDayStart": "Custom Day Start",
     "24HrClock": "24Hr Clock",
-    "clockInfo": "HabitRPG defaults to check and reset your Dailies at midnight each day. You can customize that here (Enter number between 0 and 24).",
+    "customDayStartInfo": "HabitRPG defaults to checking and resetting your Dailies at midnight each day. You can customize that here. Enter a number between 0 and 24. It is recommended that you first read <a href=\"http://habitrpg.wikia.com/wiki/Custom_Day_Start\" target=\"_blank\">this advice about how to change it</a>.",
```

Comment:
I normally recommend against changing the keys like this, but in this case I thought it was important. With this PR, we no longer have the "highly-experimental" warning on the page, but it is still recommended that people follow the  guidelines at the link, and changing the key ensures that all non-English translations will display the new text with the link immediately.